### PR TITLE
Dataminer.services instead of dataminer cloud

### PR DESCRIPTION
**Pull request by MariekeGO**
As a first step in the change from "DataMiner Cloud Services" to "dataminer.services", I've made the necessary changes to the existing DCP page in the DataMiner Overview section. @bfsantos, could you take a look and let me know if this is OK? 
Further changes to the rest of the documentation will follow later.

### CODE REVIEW

File: dataminer-overview/DataMiner_Cloud_Platform/Overview_DCP.md [hunk 1]
```diff
@@ -2,31 +2,29 @@
...
 - [Cloud-connected DataMiner Systems](#cloud-connected-dataminer-systems)
 - [DataMiner Dojo (a.k.a. Community)](#dataminer-dojo-community)
 - [DataMiner Project Collaboration](#dataminer-project-collaboration)
-- [DataMiner Cloud Services](#dataminer-cloud-services)
+- [DataMiner Catalog](#dataminer-catalog)
+- [DataMiner Live Sharing Service](#dataminer-live-sharing-service)
+- [Online integration and collaboration through Microsoft Teams](#online-integration-and-collaboration-through-microsoft-teams)
 
-The services are bundled into different tiers to which users can subscribe. Each tier contains a set of functionalities and capacities. For more information, see [DataMiner Cloud Services](https://community.dataminer.services/dataminer-cloud-platform-services/).
+The services are bundled into different tiers to which users can subscribe. Each tier contains a set of functionalities and capacities. For more information, see [dataminer.services tiers](https://community.dataminer.services/dataminer-cloud-platform-services/).
```
Comment (bfsantos) [inline dataminer-overview/DataMiner_Cloud_Platform/Overview_DCP.md:16]: this sentence can be removed, it only applies to perpetual-licensing.

File: dataminer-overview/DataMiner_Cloud_Platform/Overview_DCP.md [hunk 2]
```diff
@@ -2,31 +2,29 @@
...
 
-# DataMiner Cloud Platform
+# dataminer.services
 
-The DataMiner Cloud Services represent a genuine paradigm shift, offering innovative and game-changing solutions, which support the transition into the emerging ICT media and broadband landscape and empower digital transformation programs.
-
-The name **DataMiner Cloud Services** refers to the combined overall cloud ecosystem, which consists of different components:
+With dataminer.services, you gain access to innovative and game-changing solutions that support the transition into the emerging ICT media and broadband landscape and empower digital transformation programs. These services include the following components:
 
 - [Cloud-connected DataMiner Systems](#cloud-connected-dataminer-systems)
```
Comment (bfsantos) [inline dataminer-overview/DataMiner_Cloud_Platform/Overview_DCP.md:9]: Remote access - it refers to this topic https://docs.dataminer.services/user-guide/Cloud_Platform/RemoteAccess/Cloud_Remote_Access.html

File: dataminer-overview/DataMiner_Cloud_Platform/Overview_DCP.md [hunk 3]
```diff
@@ -2,31 +2,29 @@
...
 > [!TIP]
-> For more information on how to access the DataMiner Cloud Platform, see [DataMiner Cloud Services](https://skyline.be/dataminer/cloud-services).
+> For more information on how to access dataminer.services, see [DataMiner Services](https://skyline.be/dataminer/cloud-services).
 
 ![Cloud services](~/dataminer-overview/images/CC_cloud_services.png)
 
 ## Cloud-connected DataMiner Systems
 
-By making your DataMiner System cloud-connected, i.e. connecting it to the DataMiner Cloud Platform via a single and secured connection, you gain a host of additional capabilities (e.g. ChatOps, dashboard sharing, and custom applications).
+By connecting your DataMiner System to the cloud via a single and secured connection, you gain access to dataminer.services and with that to a host of additional capabilities (e.g. ChatOps, dashboard sharing, and custom applications). This opens up a new world of possibilities with respect to how DataMiner integrations are done and how interactions with partners, customers, and technology vendors take place.
```
Comment (bfsantos) [inline dataminer-overview/DataMiner_Cloud_Platform/Overview_DCP.md:25]: This line can be moved to the intro (line 8 above) .

File: dataminer-overview/DataMiner_Cloud_Platform/Overview_DCP.md [hunk 4]
```diff
@@ -2,31 +2,29 @@
...
+- [Online integration and collaboration through Microsoft Teams](#online-integration-and-collaboration-through-microsoft-teams)
 
-The services are bundled into different tiers to which users can subscribe. Each tier contains a set of functionalities and capacities. For more information, see [DataMiner Cloud Services](https://community.dataminer.services/dataminer-cloud-platform-services/).
+The services are bundled into different tiers to which users can subscribe. Each tier contains a set of functionalities and capacities. For more information, see [dataminer.services tiers](https://community.dataminer.services/dataminer-cloud-platform-services/).
 
 > [!TIP]
-> For more information on how to access the DataMiner Cloud Platform, see [DataMiner Cloud Services](https://skyline.be/dataminer/cloud-services).
+> For more information on how to access dataminer.services, see [DataMiner Services](https://skyline.be/dataminer/cloud-services).
 
 ![Cloud services](~/dataminer-overview/images/CC_cloud_services.png)
```
Comment (bfsantos) [inline dataminer-overview/DataMiner_Cloud_Platform/Overview_DCP.md:21]: Remote access

File: dataminer-overview/DataMiner_Cloud_Platform/Overview_DCP.md (hunk 3 above)
Comment (bfsantos) [reply dataminer-overview/DataMiner_Cloud_Platform/Overview_DCP.md:25]: "By connecting your DataMiner System to dataminer.services via a single..."

**Comment by MariekeGO** [comment]
@bfsantos, I've made the necessary changes based on your feedback. Could you take another look?

**Review by bfsantos — APPROVED** [review]

**Pull request merged**